core/flyout_button.js

```diff
@@ -236,6 +243,13 @@ Blockly.FlyoutButton.prototype.getPosition = function() {
   return this.position_;
 };
 
+/**
+ * @returns {string} Text of the button.
+ */
+Blockly.FlyoutButton.prototype.getText = function() {
```

Comment:
Can we use something more specific than `getText`, perhaps `getLabelText` or `getButtonText`. I only ask as we've traditionally use text liberally in things like fields, and it became confusing what it referred to.